contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py

```diff
@@ -131,7 +132,10 @@ def _create_envelope(self, data_point, timestamp, properties):
             tags=dict(utils.azure_monitor_context),
             time=timestamp.isoformat(),
         )
-        envelope.name = "Microsoft.ApplicationInsights.Metric"
+        if self._is_stats:
+            envelope.name = "Statsbeat"
+        else:
+            envelope.name = "Microsoft.ApplicationInsights.Metric"
```

Comment:
You can save payload by sending "Metric" directly for regular custom metrics.

Reply:
Possibly. The benefit of saving the payload doesn't really outweigh the need to change unrelated code.